src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -246,6 +287,7 @@ public void insertRecordToBuffer(SinkRecord kafkaSinkRecord) {
       bufferLock.lock();
       try {
         this.streamingBuffer.insert(kafkaSinkRecord);
+        LOGGER.debug("Record with offset [{}] inserted into buffer", kafkaSinkRecord.kafkaOffset());
```

Comment:
do we need to say which partition is this?

Reply:
Actually it's only used for debugging. I will delete this log line.